### ISSUE TITLE
Merge with official Omada integration in HA 2023 beta?

### ISSUE THREAD

**Issue by ankushg** [zachcheatham/ha-omada#60]
Looks like they're adding an official Omada integration: ttps://github.com/home-assistant/core/pull/81223

@MarkGodwin worked on it, but it looks like it might actually be lacking in features compared to this one 

Maybe there's opportunity to unify the two projects?

**Comment by zachcheatham** [comment]
Skimming through that PR, there are enough fundamental differences to where I'm not sure if its worth trying to merge them. I'm just glad the domains aren't going to conflict. 

I will continue to add features and maintain this project for the foreseeable future.

**Comment by MarkGodwin** [comment]
I had to start very small with the new integration to meet the rules for the HA PR process. The plan is to add many more features over time, but to start with I went with a feature that this integration didn't have. I started the process in October, so it took a long time to get it integrated. Hopefully new feature PRs will come through faster.
I don't want to tread on your toes, but I'm happy to do the majority of the work to bring across the missing features. I'm happy to open up to more maintainers if there is enough interest.
